media-video/webcamoid/webcamoid-9999.ebuild

```diff
@@ -50,18 +49,13 @@ RDEPEND="${COMMON_DEPEND}
 "
 
 src_configure() {
-	#Disable git in package source. If not disabled the cmake configure process will show a lot of "fatal not a git repository" errors
-	sed -i 's|find_program(GIT_BIN git)|#find_program(GIT_BIN git)|' libAvKys/cmake/ProjectCommons.cmake || die
-
 	local mycmakeargs=(
 		"-DNOMEDIAFOUNDATION=1"
-		"-DNOAVFOUNDATION=1"
```

Comment:
NOAVFOUNDATION and NOCOREAUDIO are no longer being used so it just results in a cmake warning when using these flags